### DISCUSSION TITLE
Primi passi e qualche dubbio

### DISCUSSION THREAD

**Discussion by JDrillz** [indomus/forum#2923] [GENERALE - Discussioni generiche]
Ciao a tutti, mi sto approcciando adesso al mondo della domotica, e avrei bisogno di qualche consiglio.
Premetto che ho già letto parecchio negli ultimi giorni, inclusa la guida sul sito.

Attualmente le due cose molto semplici che vorrei fare sono l'accensione automatica di una luce all'ingresso di casa quando viene aperta la porta, e l'accensione dei led dietro alla tv solo quando questo è acceso e solo a determinati orari o condizioni di visibilità.
Tuttavia nei piani c'era già l'idea di aumentare le automatizzazioni, come quelle delle tapparelle, delle tende ecc..., quindi ho iniziato ad informarmi bene per iniziare questo percorso con un approccio "future proof".

All'inizio, come penso molti di voi, mi ero subito fatto abbagliare dalla semplicità di integrazione del wifi, quindi mi ero detto "prendo due sonoff, uno per l'interruttore dell'ingresso e uno per la presa dei led della tv e via". Poi però ho letto che il wifi non è particolarmente consigliabile (quantomeno non con l'idea di ampliamento) e che forse era meglio andare su shelly piuttosto che su sonoff... però shelly è su cloud proprietario quindi meglio di no... insomma le cose si sono ben presto complicate.

Ho capito che thread sarebbe preferibile praticamente in tutto, ma poiché c'è pochissima roba che si basa su quello, ed escludendo il wifi per le ragioni che sappiamo, sono rimaste due alternative: Z-Wave e Zigbee. Sarei più orientato verso Zigbee, perché mi pare di capire che sia più diffuso (a livello di community) un pelo più economico da integrare, molto apprezzato, e abbia latenze più basse. Il fattore interferenze pare sia abbastanza trascurabile o comunque aggirabile.

Tornando indietro agli attuatori, Shelly che io sappia funziona solo tramite wifi o con z-wave, mentre sonoff solo con wifi e con zigbee, il che escluderebbe i primi.
Però qua salta fuori la prima domanda: a parte i sonoff ZBMINI-L2 quali altri attuatori, magari preferibili, ci sono compatibili con zigbee? Non ho niente contro sonoff, ma vorrei capire se sono la scelta migliore.

Come HUB utilzzerò il mio nas synology con home assistant (sì lo so, se decido di andare su zigbee/z-wave mi serve il ricevitore usb).
Avevo pensato anche di partire già con qualcosa di Thread, anche per risparmiarmi l'acquisto del ricevitore, ma sinceramente non sono riuscito a trovare attuatori compatibili (il minir4m come saprete meglio di me è compatibile con matter over wifi).

Scusate se mi sono dilungato, ma volevo dare il quadro completo, grazie a tutti per l'attenzione!

**Comment by KC-inDomus** [comment]
sugli attuatori puoi andare sernamente di wifi, cosa francamente preferibile perché, a differenza di altri standard come thread, zigbee, zwave, ha un single point of failure in meno (e sull'attuazione francamente prerisco ad averne meno possibile). sui sensori altro discorso, per ora stai su zigbee, è iper adottato, si trova di tutto, integri di tutto via home assistant.

**Reply by JDrillz** [reply]
Grazie per la risposta, non mi aspettavo di riceverne una così velocemente.

Ho un dubbio però: gli attuatori di solito sono il componente più numeroso in un impianto, se volessi domotizzare ogni singola presa ed interruttore non sarebbe difficile arrivare a qualche decina. Questo non crea un problema per quanto riguarda il numero di dispositivi che la rete wifi dovrebbe poi gestire? Al di là della quantità di dati irrisoria che richiede un attuatore, parlo proprio di numero di dispositivi collegati contemporaneamente.

**Reply by KC-inDomus** [reply]
ti rispondo dicendoti che in casa mia ha oltre 50 host wi-fi senza che la rete abbia particolari problemi di gestione o performance :)

**Reply by JDrillz** [reply]
Del fattore "always on" dei dispositivi wifi che mi dici però? Avere attuatori costantemente online non è controproducente alla lunga? Va bene che sono consumi minimi, ma vanno sempre moltiplicati per tutti i dispositivi.

Ad ogni modo, supponendo quindi di restare sul wifi, saresti team Shelly o Sonoff?
Gli Shelly sono abbastanza più costosi (un MINIR4M lo trovo a circa 15€ spedito, mentre con Shelly, applicando le spese di spedizione, un Plus 1 Mini Gen3 viene a costare poco più di 21€, che non è una differenza di poco conto ripetuta per n dispositivi) e si basano sul cloud proprietario, però è anche vero che utilizzando HA lo bypasserei. Tra l'altro PARE che diano meno problemi proprio su HA rispetto ai Sonoff.
I Sonoff sono quindi più economici (la versione non matter costa altri 2€ meno volendo) e dovrebbero avere una morsettiera migliore, ma appunto PARE più noiosi da configurare su HA.

**Reply by KC-inDomus** [reply]
comunque sia, anche eventuali attuatori zigbee (o altro) andrebbero comunque alimentati in continuo.
personalmente preferisco sonoff, ma anche gli shelly vanno super bene, specialmente su carichi non troppo importanti. indubbiamente, shelly si integra meglio di sonoff (almeno, out of the box) con HA.

**Reply by JDrillz** [reply]
Cosa intendi dicendo che si integra meglio? Non rileva i sonoff in automatico o che?

Grazie

**Reply by KC-inDomus** [reply]
le integrazioni shelly sono svariate e tutte molto più articolate di quella, unica, itead

**Reply by JDrillz** [reply]
Allora come mai preferisci sonoff?
Scusa le tante domande, ma sto cercando di capire più cose possibili ':)

**Reply by KC-inDomus** [reply]
costruttivamente parlando preferisco sonoff, che sono (mia percezione) disegnati e realizzati meglio; in termini di integrabilità e gestione shelly senza storia: puoi integrarli davvero in molti modi diversi, e tutti molto efficaci. in questo sonoff un po' claudica.

**Reply by JDrillz** [reply]
Capito, grazie per i chiarimenti :)
Effettivamente ho visto sul sito di HA che fra i marchi diciamo "nativamente compatibili" compare shelly ma non sonoff/itead.

**Reply by KC-inDomus** [reply]
no ma sai quello è relativo. prendi meross: non ha un componente "ufficiale" ma l'integrazione custom funziona da dio. è proprio itead che secondo me non offre un set api o cmq di interfaccimenti che siano all'altezza. tutto e solo via cloud, per ora.

**Reply by JDrillz** [reply]
Speriamo si adeguino alla svelta allora :)
Approfitto per un'altra cosa credo veloce.
Su HA ho due TV che vengono rilevate con homekit, con "android TV remote" e come "Google cast", ma qual è la differenza fra i 3 a livello di funzionalità? Ce n'è una migliore? Vorrei stare alla larga dalla roba Apple di cui fa parte home kit per l'appunto.
